src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import './style.scss';
+
+export default function App() {
+  return (
+    <React.Fragment>
```

Comment:
If you don't need to pass any prop to Fragment you can just use \<\>\</\> instead of \<React.Fragment\>